### PR TITLE
my first commit

### PR DESCRIPTION
**Pull request by snowwhitedev**
My login page work

### CODE REVIEW

File: .env copy [hunk 1]
```diff
@@ -0,0 +1,4 @@
+BASE_URL=http://192.168.10.30
```
Comment (mariapf) [inline .env copy:1]: change name maybe to .env.test instead .env copy

File: pages/login.vue [hunk 2]
```diff
@@ -1,60 +1,107 @@
...
     // Before loading login page, obtain csrf cookie from the server.
     this.$axios.$get('/sanctum/csrf-cookie')
-    console.log(this.$route.params.slug)
+    console.log('slug', this.$route.params.slug)
   },
   methods: {
     async login() {
-      this.errro = {}
+      this.$router.push({
+        path: '/2'
```
Comment (mariapf) [inline pages/login.vue:99]: @snowwhitedev  what exaclty means this /2?

File: .env copy (hunk 1 above)
Comment (snowwhitedev) [reply .env copy:1]: Okay, I see. I will remove it. It is back up of .env

File: pages/login.vue (hunk 2 above)
Comment (snowwhitedev) [reply pages/login.vue:99]: I just wanted to go to page after clicking login. I removed it already and used slug.

**Pull request closed without merging**